### PR TITLE
guides: add synth records to resource document

### PR DESCRIPTION
**Pull request by pinheadmz**
closes https://github.com/handshake-org/handshake-org.github.io/issues/133

### CODE REVIEW

File: src/guides/resource-records.md [hunk 1]
```diff
@@ -204,6 +204,99 @@ Example:
...
+> dig @127.0.0.1 -p 5349 _400hjs000l2gol000fvvsc9cpg._synth. AAAA +short
+2001:19f0:5:450c:5400:3ff:fe31:2ccc
+```
+
+Note that decoding the synth domain does not require any "lookups" of any kind.
+The IP address is decoded from the base32 string and returned as a DNS answer.
+
+#### `SYNTH6`
+
+The `SYNTH6` Handshake resource record is the same as `GLUE4` but with IPv6.
```
Comment (rithvikvibhu) [inline src/guides/resource-records.md:296]: same as `SYNTH4`* ?

Comment (pinheadmz) [reply src/guides/resource-records.md]: Thank you! 🤝  fixed

File: src/guides/resource-records.md [hunk 2]
```diff
@@ -204,6 +204,99 @@ Example:
...
+;; MSG SIZE  rcvd: 152
+
+```
+
+Handshake root name servers SHOULD support direct querying of synth domains:
+
+```
+> dig @127.0.0.1 -p 5349 _400hjs000l2gol000fvvsc9cpg._synth. AAAA +short
+2001:19f0:5:450c:5400:3ff:fe31:2ccc
+```
```
[the remark this answers is not in the document]
Comment (NetOpWibby) [reply src/guides/resource-records.md:289]: Otherwise, this PR looks good to me!

[the remark this answers is not in the document]
Comment (pinheadmz) [reply src/guides/resource-records.md:289]: awesome thanks, will fix and merge

**Pull request merged**